MERN/Topic/Backend/MongoDB/1.1 What is MongoDB(V).md

```diff
@@ -0,0 +1,12 @@
+# What is Node.js?

```

Comment:
Looks good to me ... just ~Node.js~ MongoDB :)

Reply:
Made requested changes.